Services/JenkinsDocker/Scripts/install.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+nodeName=`azure servicefabric node show | tail -n +3 | head -n -1 | cut -f2-3 -d':' |grep "name" | head -1 | cut -f2 -d':' | tr -d " ',"` 
+echo Installing Jenkins Container application on :$nodeName
+#azure servicefabric application package copy JenkinsSF fabric:ImageStore
+#azure servicefabric application type register JenkinsSF
```

Comment:
typo.. I was testing the shell-script code.. forgot to remove the comment. updated.